charts/knative-serving/Chart.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v2
+name: knative-serving
+description: A Helm chart for Knative Serving on Kubernetes
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 1.0.1
```

Comment:
The chart's version shouldn't be the same as appVersion.
The chart version can be set to v0.1.0

Reply:
I also set verison to 0.1.0 before, but considering the possibility of supporting lower versions of the component later, I modified version to adapt to appVersion

Reply:
I don't understand.
version is helm chart's version
The helm chart install will support knative v1.0.1+ starting from k8s 1.20, lower version k8s doesn't have helm install method.

![image](https://user-images.githubusercontent.com/18525465/171321326-bc1382b5-8629-4c94-bdd7-c8e14a5eb021.png)